pykafka/rdkafka/simple_consumer.py

```diff
@@ -0,0 +1,239 @@
+from contextlib import contextmanager
+import logging
+
+from pykafka.exceptions import RdKafkaStoppedException, ConsumerStoppedException
+from pykafka.simpleconsumer import SimpleConsumer, OffsetType
+from pykafka.utils.compat import get_bytes
+from . import _rd_kafka
+
+
+log = logging.getLogger(__name__)
+
+
+class RdKafkaSimpleConsumer(SimpleConsumer):
+    """A pykafka.SimpleConsumer with librdkafka-based fetchers
+
+    This aims to conform to the SimpleConsumer interface as closely as
+    possible.  There are some notable differences:
+
+    1. rotating over partitions: while message ordering within partitions is
+       conserved (of course!), the order in which partitions are visited will
+       deviate.  In particular, here we may emit more than one message from
+       the same partition before visiting another
+    2. ignores num_consumer_fetchers: librdkafka will typically spawn at least
+       as many threads as there are kafka cluster nodes
+
+    For an overview of how configuration keys are mapped to librdkafka's, see
+    _mk_rdkafka_config_lists.
+    """
```

Comment:
I've put further annotations and some improvements in the configuration mapping. It's not literally a docstring, but personally I'd feel that duplicating the mapping into a docstring (that's not code) would eventually leave it stale. If I can make readability improvements to the actual mapping in `_mk_rdkafka_config_lists` instead, so that that is more self-documenting, I think I'd prefer that option.